apps/api/src/domain/domainModelValidators/__tests__/domainModelValidators/buildDomainModelValidatorTestCases/bibliographicReferenceInvalidTestCases/bookBibliographicReference.domainModelValidatorTestCases.ts

```diff
@@ -0,0 +1,138 @@
+import BookBibliographicReferenceData from 'apps/api/src/domain/models/bibliographic-reference/entities/BookBibliographicReferenceData';
+import { DTO } from 'apps/api/src/types/DTO';
+import { EntityId } from '../../../../../../domain/types/ResourceId';
+import { resourceTypes } from '../../../../../../domain/types/resourceTypes';
+import { InternalError } from '../../../../../../lib/errors/InternalError';
+import { BookBibliographicReference } from '../../../../../models/bibliographic-reference/entities/book-bibliographic-reference.entity';
+import { BibliographicReferenceType } from '../../../../../models/bibliographic-reference/types/BibliographicReferenceType';
+import InvalidEntityDTOError from '../../../../errors/InvalidEntityDTOError';
+import { DomainModelValidatorInvalidTestCase } from '../../../types/DomainModelValidatorTestCase';
+import getValidBibliographicReferenceInstanceForTest from '../utils/getValidBibliographicReferenceInstanceForTest';
+
+const validDto = getValidBibliographicReferenceInstanceForTest(BibliographicReferenceType.book);
+
+const buildTopLevelError = (id: EntityId, innerErrors: InternalError[]): InternalError =>
+    new InvalidEntityDTOError(resourceTypes.bibliographicReference, id, innerErrors);
+
+export const buildBookBibliographicReferenceTestCases =
+    (): DomainModelValidatorInvalidTestCase<BookBibliographicReference>[] => [
+        {
+            description: 'The title is an empty string',
+            invalidDTO: {
+                ...validDto,
+                data: {
+                    ...validDto.data,
+                    title: '',
+                } as DTO<BookBibliographicReferenceData>,
+            },
+            expectedError: buildTopLevelError(validDto.id, []),
+        },
+        {
+            description: 'The creators array is empty',
+            invalidDTO: {
+                ...validDto,
+                data: {
+                    ...validDto.data,
+                    creators: [],
+                    // TODO remove cast
+                } as unknown as DTO<BookBibliographicReferenceData>,
```

Comment:
Let's see if we can remove this cast. We'll need to make `getValidBibliographicReferenceInstanceForTest` generic. We'll probably want a mapped type `BibliographicReferenceDataTypeToModel` and use type inference, in which case, we probably won't actually have to pass a type to the generic on this side.